Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -144,4 +210,88 @@ extension ServiceRunnable {
             .setFailureType(to: Error.self)
             .eraseToAnyPublisher()
     }
+
+    func generateBoundary() -> String {
+        return UUID().uuidString
+    }
+
+    // swiftlint:disable line_length cyclomatic_complexity
```

Comment:
Why disable line_length?

Reply:
The append functions look easier to read without line breaks, but I can remove the rule and add line breaks